cigaradvisor/scripts/scripts.js

```diff
@@ -248,21 +248,27 @@ export function getRelativePath(path) {
   return relPath;
 }
 
-let articleIndexData;
-
+const articleIndexData = [];
 /**
  * Returns all the posts in the posts index.
  *
  * @return {Promise<Array[Object]>}
  */
-export async function loadPosts() {
-  if (!articleIndexData) {
-    const resp = await fetch(ARTICLE_INDEX_PATH);
+export async function loadPosts(path = ARTICLE_INDEX_PATH, flag = false) {
```

Comment:
What does `flag` mean?

Reply:
flag is set to true for subsequent calls when there are more articles than the limit.

Reply:
the name of the parameter should reflect it's intent/use. 'flag' doesn't convey any context.